### DISCUSSION TITLE
Automatically assign Serial Numbers

### DISCUSSION THREAD

**Discussion by Guusggg** [inventree/InvenTree#2461] [General]
Hi everybody,

I vaguely remember there being an option in the inventree settings to automatically assign Serial Numbers. I couldn't find it today, any help you guys can offer me?

Thanks!

**Comment by matmair** [comment]
Not totally sure if we are talking about the same. The option to use the syntax 8+ for: use serial numbers from 8 till all parts have a number should still work.

**Reply by Guusggg** [reply]
Ah okay, I did understand the syntax but the field where you can input specifies the "next serial number" which I would like to use, as we use counting up as our serial numbers. I wanted to copy the next number, but it is in the placeholder of the text field. I just want to use the next available serial number. Is this possible with the `+` syntax? If I input `1+`, the serial number becomes 1, which is not the same as the next available serial number (which is 202010+)

**Reply by matmair** [reply]
@Guusggg I do not think this works right now. Maybe that would be a suggestion for a FR?

**Reply by Guusggg** [reply]
@matmair Will do! Thanks

**Reply by eeintech** [reply]
Fixed in #2487 right? Mark as answered?

**Reply by matmair** [reply]
@eeintech general discussions do not have answers, only stuff marked as q&a has

**Reply by eeintech** [reply]
Oh okay my bad!
